### PR TITLE
Update basic-stateful-set.md

### PR DESCRIPTION
**Pull request by zhangbei59**
Modify nslookup query output format  in en doc

### CODE REVIEW

File: content/en/docs/tutorials/stateful-application/basic-stateful-set.md [hunk 1]
```diff
@@ -270,7 +270,13 @@ Address 1: 10.0.0.10 kube-dns.kube-system.svc.cluster.local
...
 Address 1: 10.244.1.7
-
+```
+```shell
+#  And run this in the dns-test container shell
+nslookup web-1.nginx
+```
+The output is similar to:
+```
 nslookup web-1.nginx
```
Comment (tengqm) [inline content/en/docs/tutorials/stateful-application/basic-stateful-set.md:280]: remove this line

**Review by zhangbei59 — COMMENTED** [review]
ok

**Review by kundan2707 — COMMENTED** [review]
/lgtm

**Comment by sftim** [comment]
Mindful of https://github.com/kubernetes/website/pull/40765#discussion_r1170829028:

/lgtm cancel

File: content/en/docs/tutorials/stateful-application/basic-stateful-set.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/tutorials/stateful-application/basic-stateful-set.md:280]:
```suggestion
```

**Review by sftim — COMMENTED** [review]
You should make the recommended change @zhangbei59 

A maintainer may close this PR if we don't see progress there.

**Comment by sftim** [comment]
No progress since https://github.com/kubernetes/website/pull/40765#pullrequestreview-1520698223

/close

**Pull request closed without merging**